lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
 import ImportStatement from './ImportStatement';
 
+const STYLE_SIDE_EFFECT = 'side-effect';
```

Comment:
Same comment about naming here. What do you think about `STYLE_SIDE_EFFECT_ONLY`/`side-effect-only`?